stl/inc/ranges

```diff
@@ -4930,25 +4930,28 @@ namespace ranges {
                 return _Left._Current <=> _Right._Current;
             }
 
-            _NODISCARD_FRIEND constexpr _Iterator operator+(_Iterator _It, const difference_type _Off) noexcept(
-                noexcept(_It._Current += _Off)) /* strengthened */
+            _NODISCARD_FRIEND constexpr _Iterator operator+(const _Iterator& _It, const difference_type _Off) noexcept(
+                noexcept(_STD declval<_Iterator>() += _Off)) /* strengthened */

```

Comment:
This... isn't _actually_ a problem, since we don't reference qualify `_Iterator::operator+=`, but I'd prefer if we used `_Iterator&` here.